wrappers/services.go

```diff
@@ -130,7 +130,20 @@ func StartServices(apps []*snap.AppInfo, inter interacter) (err error) {
 		}(app)
 
 		if len(app.Sockets) == 0 && app.Timer == nil {
-			services = append(services, app.ServiceName())
+			// check if the service is disabled, if so don't start it up
+			// this could happen for example if the service was disabled in
+			// the install hook by snapctl
+			statuses, err := sysd.Status(app.ServiceName())
```

Comment:
`Status()` is pretty invovled, just a thought but maybe a thin wrapper around `systemctl is-enabled`, eg `IsEnabled(serviceName string) (bool, error)`, would be useful.

Eg:
```
// IsEnabled checkes whether the given service is enabled
func (*systemd) IsEnabled(serviceName string) (bool, error) {
	_, err := systemctlCmd("--root", s.rootDir, "is-enabled", serviceName)
	if err == nil {
		return true, nil
	}
	if sysdErr, ok := err.(*Error); ok && sysdErr.msg == "disabled" {
		return false, nil
	}
	return false, err
}
```

Reply:
I agree with Maciej here. Plus thanks to the spread test we will have nice coverage of various systemd versions across the history

Reply:
I implemented this method basically as provided with a few changes. I'm also checking the exit code to be 1, which seems to be standard behavior for `systemctl is-enabled` to be as strict as possible with ensuring that the service is actually disabled.